### DISCUSSION TITLE
Support for Apple sillicon

### DISCUSSION THREAD

**Discussion by vranicf** [tiangolo/uvicorn-gunicorn-fastapi-docker#325] [Questions] [answered]
Hi, 

Will there be support for arm64 architecture in the near future? 
Offering it on top of linux/amd64 would be perfect. Thanks in advance!

**Comment by tiangolo** [comment, marked as answer]
Now that Uvicorn supports managing workers with `--workers`, including restarting dead ones, there's no need for Gunicorn. That also means that it's much simpler to build a Docker image from scratch now, I updated the docs to explain it.

Because of that, I deprecated this Docker image: https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker#-warning-you-probably-dont-need-this-docker-image

That also means that if you build from the official base Python you would have multi-arch (including Apple) by default. :sunglasses:
